src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java

```diff
@@ -52,7 +52,8 @@ public void copiesFromCharArrayWithCharsetToFile() throws IOException {
         final String input =
             "Hello, товарищ file #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
-        MatcherAssert.assertThat(
+        new Assertion<>(
```

Comment:
@iiylll you need to call `affirm()` on all the `Assertion` objects for the assertion to be verified. Currently those tests do nothing.